src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+        <p>More details on the checksum can be found here: <a href="/sdk/hardware/addon-protocols/rs232-usb/#checksum">Add-On Protocol - RS232 &amp; USB</a></p>
+        <h4>Log Type: 2 (GenericFaultRecord)</h4>
+        <p>Typically used to log a fault condition that needs to be escalated to a supervisor for human intervention.</p>
+        <table>
+            <thead>
+                <tr>
+                    <th>Data</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>1-2</td>
+                    <td>Fault code</td>
+                </tr>
+                <tr>
+                    <td>3</td>
+                    <td>
+                        Fault = 1
+                        <br></br>
+                        Information = 0
+                    </td>
+                </tr>
+                <tr>
+                    <td>4</td>
+                    <td>
+                        Log Once Per Trip = 1
+                        <br></br>
+                        Log Every Fault = 0
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>IOX Multi-Frame Log Data (0x1E)</h3>
+        <p>Sent from the IOX to the GO device when the IOX wants create a log that cannot fit into a single CAN frame. The first frame contains the Type and Length. All frames start with a Frame Counter that is an incrementing sequence number. The first frame always starts with 0x00.</p>
+        <h4>Payload First Frame</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0</td>
+                    <td>Frame Counter (0x00)</td>
+                </tr>
+                <tr>
+                    <td>1</td>
+                    <td>Log Type</td>
+                </tr>
+                <tr>
+                    <td>2-3</td>
+                    <td>Data Length</td>
+                </tr>
+                <tr>
+                    <td>4-7</td>
+                    <td>Data</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Payload Subsequent Frames</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0</td>
+                    <td>Frame Counter</td>
+                </tr>
+                <tr>
+                    <td>1-7</td>
+                    <td>Data</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Log Types</h4>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on